akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala

```diff
@@ -16,8 +16,15 @@ import akka.util.Helpers.{ Requiring, ConfigOps, toRootLowerCase }
 import scala.concurrent.duration.FiniteDuration
 import akka.japi.Util.immutableSeq
 
-final class ClusterSettings(val config: Config, val systemName: String) {
+object ClusterSettings {
+  type Team = String
+  val TeamRolePrefix = "team-"
+  val DefaultTeam: Team = "default"
+
+}
```

Comment:
Should we make the team-as-roles encoding public or should we treat it as an implementation detail to be able to change that later on? (Maybe that has been decided already?)